src/Sylius/Bundle/PayumBundle/Form/DataTransformer/PaypalGatewayConfigurationTransformer.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\PayumBundle\Form\DataTransformer;
+
+use Symfony\Component\Form\DataTransformerInterface;
+
+/**
+ * @author Mateusz Zalewski <mateusz.zalewski@lakion.com>
+ */
+final class PaypalGatewayConfigurationTransformer implements DataTransformerInterface
```

Comment:
If it is used only once, we may make it inline using `CallbackTransformer`, it looks quite specific.